mlflow/cli.py

```diff
@@ -554,6 +564,33 @@ def gc(older_than, backend_store_uri, run_ids):
     else:
         run_ids = run_ids.split(",")
 
+    deleted_experiment_ids = []
+    next_page_token = None
+    current_time = get_current_time_millis()
```

Comment:
```suggestion
    time_threshold = get_current_time_millis() - time_delta
```

We can compute the threshold here.

Reply:
Okay, I will make the change.